Library/TimeInterval+ISO8601DateTests.swift

```diff
@@ -0,0 +1,11 @@
+@testable import Library
+import XCTest
+
+class TimeInterval_ISO8601DateTests: XCTestCase {
+  func testConvertToISO8601DateTimeString_FromTimeInterval() {
```

Comment:
`testToISO8601DateTimeString()`

Reply:
Also can you add a test case here? We should test the past and the future.

Reply:
So the function essentially converts TimeInterval to Date String, either present, past or current as `Date(timeIntervalSince1970:)` will always give you date from `TimeInterval` irrespective. I can add more of those test regardless, just wanted you to know that.

Reply:
Yup sounds good. Just want to test those distinct conditions. I think it will bump test coverage as well, which is nice.